DEPENDENCIES

```diff
@@ -98,11 +98,36 @@ Prerequisites for building the OpenDataPlane (ODP) API
      --with-openssl-path=/home/${USER}/src/install-openssl-aarch64
    $ make
 
```

Comment:
The commit messages could be improved.
- The subject lines could follow the style of other ODP commits
- The commits talk about ARMv8 crypto API, but I think a more correct reference would be ARMv8 crypto library or AArch64cryptolib.
- The first commit talks about null crypto changes but there are no changes in the null crypto implementation.

The commit message for the first commit could be something like this:

```
linux-gen: crypto: integrate Aarch64cryptolib

Integrate Aarch64cryptolib in the build using pkg-config. Add an
autoconf config option to select the crypto implementation to be used
and add armv8crypto as an alternative to openssl and null. Add a null
crypto based skeleton crypto implementation to be used with the
Aarch64cryptolib.
```

And for the second commit, something like this:
```
linux-gen: crypto: Add AES-GCM to armv8crypto

Implement AES-GCM in armv8crypto using the AArch64cryptolib.
```

Reply:
I will amend the commit messages as you have suggested. Thanks!